lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -5,12 +5,13 @@ use std::sync::Arc;
 use ain_cpp_imports::get_eth_priv_key;
 use ain_evm::bytes::Bytes;
 use ain_evm::core::EthCallArgs;
+use ain_evm::block::BlockError;
```

Comment:
Failing rust lint check here too

Reply:
Resolved in [e7668d5](https://github.com/DeFiCh/ain/pull/2324/commits/e7668d577a89d0f85127fa30efc090f635530d3c)